packages/lesswrong/components/common/Row.tsx

```diff
@@ -8,13 +8,14 @@ const styles = (theme: ThemeType) => ({
   }
 });
 
-export const Row = ({classes, justifyContent="space-between", alignItems="center", children}: {
+export const Row = ({classes, justifyContent="space-between", alignItems="center", children, gap}: {
   classes: ClassesType<typeof styles>,
   alignItems?: "flex-start" | "flex-end" | "center" | "baseline" | "stretch",
   justifyContent?: "flex-start" | "flex-end" | "center" | "space-between" | "space-around" | "space-evenly" | "stretch",
+  gap?: number,
   children: React.ReactNode
 }) => {
-  return <div className={classes.root} style={{justifyContent, alignItems}}>
+  return <div className={classes.root} style={{justifyContent, alignItems, gap: gap ? `${gap}px` : '0px'}}>
```

Comment:
I don't know what effect a new default of `gap: 0px` might have, but I think this should not include a default when `gap` isn't passed in:
```
const gapStyle = typeof gap === 'number' ? { gap: `${gap}px` } : {};
return <div ... style={{justifyContent, alignItems, ...gapStyle}}>
```